cgstats/db/models.py

```diff
@@ -158,7 +158,7 @@ class Supportparams(Model):
     program = Column(types.String(255))
     commandline = Column(types.Text)
     sampleconfig_path = Column(types.String(255))
-    sampleconfig = Column(types.Text)
+    sampleconfig = Column(types.Text(16777215))
```

Comment:
it would be nice to add a sql file with this change so we run it on the prod db:

change-sampleconfig-mediumtext.sql:
```suggestion
ALTER TABLE `supportparams` change `sampleconfig` `sampleconfig` MEDIUMTEXT;
```

Reply:
I'll fix it :|

Reply:
Done